pkg/networkserver/networkserver_flow_test.go

```diff
@@ -628,6 +628,7 @@ func handleClassAOTAAEU868FlowTest1_0_2(ctx context.Context, conn *grpc.ClientCo
 					return
 				}
 				t.Log("Uplink handling failed with a not-found error. The join-accept was scheduled, but the new device state most probably had not been written to the registry on time, retry.")
+				time.Sleep(test.Delay)
```

Comment:
This gives some time for the NS to actually write the device before retrying the same thing.
In single-threaded environments this would definitely cause problems, which may be the case in CI.